### PR TITLE
new autobuff spells for RG

### PR DESCRIPTION
**Pull request by Demnokkoyen**
This PR adds new spells for RG and IG classes.

- Spear Quicken;
- Shield Spell;
- Inspiration;
- Vanguard Force.

### CODE REVIEW

**Comment by Demnokkoyen** [comment]
Actually, Shield Spell ID is incorrect right now. Don't merge it yet, i'll fix it soon D:

**Comment by Demnokkoyen** [comment]
Ok, it should be fixed and ready to be merged now.

File: Model/Autobuff.cs [hunk 1]
```diff
@@ -43,6 +43,9 @@ public _4RThread AutoBuffThread(Client c)
                     if (buffMapping.ContainsKey(status)) //CHECK IF STATUS EXISTS IN STATUS LIST AND DO ACTION
                     {
                         bmClone.Remove(status);
+                    } else if (currentStatus <= 5000)
+                    {
+                        Console.WriteLine("Unrecognized effect ID: " + currentStatus);
```
Comment (afonsocdaniel) [inline Model/Autobuff.cs:48]: I would suggest removing this debug line.

Comment (Demnokkoyen) [reply Model/Autobuff.cs:48]: Wow, sorry! ❌

Comment (Demnokkoyen) [reply Model/Autobuff.cs]: > I would suggest removing this debug line.

Fixed :)

**Comment by markiing** [comment]
Thanks @Demnokkoyen

**Pull request merged**